filters/apiusagemonitoring/spec.go

```diff
@@ -120,6 +123,31 @@ type apiUsageMonitoringSpec struct {
 	clientKeys            []string
 	realmsTrackingMatcher *regexp.Regexp
 	unknownPath           *pathInfo
+	onPanicSometimes      rate.Sometimes
+}
+
+func (s *apiUsageMonitoringSpec) sometimesErrorf(args ...interface{}) {
+	s.onPanicSometimes.Do(func() {
+		log.Errorf(args[0].(string), args[1:]...)
+	})
+}
```

Comment:
I think it makes sense to check log.Errorf signature and use the same for this function.
Then I propose we name them simply `errorf`, `warnf` and `infof`